CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+
+If you would like to contribute but have no specific feature in mind, simply
+head over to the [issue board](https://github.com/furrer-lab/abn/issues) where
+you will find open issues  with the label
+https://github.com/furrer-lab/abn/labels/help%20wanted.
+Feel free to comment on them if it is not clear to you what exactly the issue
+is about.
+Some issues are also labeled with
+https://github.com/furrer-lab/abn/labels/good%20first%20issue, they might be
+more suitable if you are new to abn.
+
+In case you already have some specific contribution you would like to add,
+first check the [issue board](https://github.com/furrer-lab/abn/issues)
+(also see the
+[closed issues](https://github.com/furrer-lab/abn/issues?q=is%3Aissue+is%3Aclosed))
+if you can find existing issues related to the contribution you plan to make.
+If yes, leave a comment on the related issue.
+If no, create a [new issue](https://github.com/furrer-lab/abn/issues/new) describing
+the feature you want to implement.
+For some inspiration on how to write a great issue refer to the
+[tidyteam code review prinicples](https://code-review.tidyverse.org/issues/).
+
+Once the foreseen edits are documented in an issue, create a pull request
+(to see how simply follow
+[Github's instructions on creating a Pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request))
+and add your changes to the related branch.
+
+## Development
+
+We use [renv](https://rstudio.github.io/renv/index.html) for our development
+environment and would encourage you to do the same.
+Therefore, abn directly provides a development environment and we suggest the
+following workflow for development:
+
+*   Fork the package and clone it onto your computer.
+    
+    _Note:_ You might do this with `usethis::create_from_github("furrer-lab/abn", fork = TRUE)`.
+
+*   Install `renv` with `install.packages("renv")`
+*   Restore the development environment with `renv::restore()`
+*   Create a new branch for your pull request.
+    
+    _Note:_ You might use `usethis::pr_init("explicative-branch-name")`.
+    
+    _See [Testing](#testing) for flags you can set in the branch name_
+
+*   Edit the code and commit your changes.
+  
+    _Note:_ With `usethis` you would run: `usethis::pr_push()` and the follow 
+    the prompts.
+    The title of your pull request should briefly describe the change.
+    The body of your pull request should contain `Closes #issue-number`, where
+    `#issue-number` is the identifier of the related issue.
+
+## Testing
+
+The abn package uses several testing pipelines.
+When developing (i.e. not on the `main` branch) we recommend running the
+[quick-testthat.yml](https://github.com/furrer-lab/abn/blob/24-documentation-of-the-testing-procedure-noT/.github/workflows/quick-testthat.yml) action.
+This pipeline runs be default on push event on all branches other than `main`
```

Comment:
typo be -> by

Reply:
https://github.com/furrer-lab/abn/pull/82/commits/d6f86b752e310bec5998efb4bd6eb9077ac0e097